src/Microsoft.DncEng.SecretManager/Commands/SynchronizeCommand.cs

```diff
@@ -84,6 +85,15 @@ public override async Task RunAsync(CancellationToken cancellationToken)
             foreach (var (name, secret, secretType, secretReferences) in orderedSecretTypes)
             {
                 _console.WriteLine($"Synchronizing secret {name}, type {secret.Type}");
+
+                if (existingSecrets.TryGetValue(name, out var existingSecret))
+                {
+                    if (!existingSecret.Tags.ContainsKey(AzureKeyVault.NextRotationOnTag))
+                    {
+                        _console.LogError($"Key Vault Secret '{name}' is missing {AzureKeyVault.NextRotationOnTag} tag, using the end of time as value. Please force a rotation or manually set this value.");
```

Comment:
Does the `using the end of time as value` part still apply?

Reply:
it does, later we're going through the SecretProperties and using the end of time value for those that don't have the `NextRotationOnTag`

Reply:
True, I improved the logging a bit